include/stronk/stronk.h

```diff
@@ -374,6 +374,82 @@ struct can_hash
 template<typename StronkT>
 concept can_hash_like = std::same_as<typename StronkT::can_hash_indicator, std::true_type>;
 
+template<typename StronkT>
+struct can_size
+{
+    [[nodiscard]] constexpr auto size() const noexcept -> std::size_t
+    {
+        return static_cast<const StronkT&>(*this).template unwrap<StronkT>().size();
+    }
+    [[nodiscard]] constexpr auto empty() const noexcept -> bool
+    {
+        return this->size() == 0ULL;
```

Comment:
How will this behave on a 32-bit system? :-)

Reply:
I think it will work?

Reply:
https://stackoverflow.com/a/65934742 <- in 23 we get size_t literal.